src/style.css

```diff
@@ -0,0 +1,160 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: serif, monospace;
+}
+
+input {
+  padding: 0.5rem;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin: auto;
+  width: 40%;
+  min-width: 500px;
+  min-height: 500px;
+  background-color: #fff;
+  box-shadow: 0 0 20px #666;
+  border-radius: 2px;
+  position: absolute;
+  top: 20vh;
+  left: 30vw;
+}
```

Comment:
kindly, I suggest separate property `display: flex` to a single class called `.d-flex { display: flex; }`

kindly separate property `justify-content: space-between;` to a single class called `.space-between {
    justify-content: space-between;
}` 

kindly if u repeated it below  remove it and update HTML file 

Don't forget update HTML file: `<div class="d-flex space-between container">`